src/main/java/de/tum/in/www1/artemis/service/CourseService.java

```diff
@@ -271,6 +274,107 @@ public void registerUserForCourse(User user, Course course) {
         log.info("User " + user.getLogin() + " has successfully registered for course " + course.getTitle());
     }
 
+    /**
+     * Fetches Course Management data from repository and returns a list of Course DTOs
+     *
+     * @param isOnlyActive Whether or not to include courses with a past endDate
+     * @return A list of Course DTOs for the course management overview
```

Comment:
It seems to return a course entity and not a DTO unless I am missing something here

Reply:
👍